manager/controllers/cves/details.go

```diff
@@ -0,0 +1,81 @@
+package cves
+
+import (
+	"app/base/logging"
+	"app/base/models"
+	"app/base/utils"
+	"app/manager/base"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type GetCveDetailsSelect struct {
+	Cvss2Score    *float32         `json:"cvss2_score"`
+	Cvss2SMetrics *string          `json:"cvss2_metrics"`
```

Comment:
Cvss2**S**Metrics  , probably a typo it should be Cvss2Metrics.

Reply:
@yungbender fixed